packages/lib-webpack/src/webpack/DynamicRemotePlugin.ts

```diff
@@ -134,22 +158,24 @@ export class DynamicRemotePlugin implements WebpackPluginInstance {
       );
     }
 
-    if (compiler.options.output.uniqueName) {
+    if (this.libraryType === 'jsonp' && compiler.options.output.uniqueName) {
```

Comment:
Why does this apply only to `jsonp` option?

Reply:
Is is a left over when there was the `plugin-entry.js` default name. With the hash changes, it can be removed.